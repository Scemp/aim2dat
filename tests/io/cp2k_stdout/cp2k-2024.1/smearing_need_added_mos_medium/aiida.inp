&GLOBAL
  PROJECT aiida
  RUN_TYPE ENERGY
  PRINT_LEVEL MEDIUM
&END GLOBAL
&FORCE_EVAL
  METHOD QS
  &SUBSYS
    &CELL
      ABC 2.950 2.950 2.950
      ALPHA_BETA_GAMMA 60.0 60.0 60.0 
      PERIODIC XYZ
    &END CELL
    &COORD
      Au 0.000 0.000 0.000
    &END COORD
    &KIND Au
      BASIS_SET DZVP-MOLOPT-SR-GTH-q11
      POTENTIAL GTH-PBE-q11
    &END KIND
  &END SUBSYS
  &DFT

    BASIS_SET_FILE_NAME BASIS_MOLOPT
    BASIS_SET_FILE_NAME BASIS_MOLOPT_UCL
    POTENTIAL_FILE_NAME GTH_POTENTIALS
    &MGRID
      NGRIDS 4
      CUTOFF 400
      REL_CUTOFF 40
    &END MGRID
    &QS
      EPS_DEFAULT 1.0E-10
    &END
    &POISSON
      PERIODIC XYZ
    &END POISSON
    &SCF
      ADDED_MOS 0

      SCF_GUESS RESTART
      MAX_SCF 150
      EPS_SCF 1.0E-6
      &MIXING T
        METHOD BROYDEN_MIXING
        NBUFFER 20
        ALPHA 0.3
      &END MIXING

      &SMEAR
        METHOD FERMI_DIRAC
        ELECTRONIC_TEMPERATURE 300
      &END SMEAR
    &END SCF
    &KPOINTS
      SCHEME MONKHORST-PACK 2 2 2
      PARALLEL_GROUP_SIZE 0
    &END KPOINTS
    &XC
      &XC_FUNCTIONAL PBE
      &END XC_FUNCTIONAL
    &END XC
  &END DFT
&END FORCE_EVAL


