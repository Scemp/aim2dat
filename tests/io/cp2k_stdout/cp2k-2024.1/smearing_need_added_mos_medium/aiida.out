 DBCSR| CPU Multiplication driver                                           XSMM (U)
 DBCSR| Multrec recursion limit                                              512 (U)
 DBCSR| Multiplication stack size                                           1000 (D)
 DBCSR| Maximum elements for images                                    UNLIMITED (U)
 DBCSR| Multiplicative factor virtual images                                   1 (U)
 DBCSR| Use multiplication densification                                       T (D)
 DBCSR| Multiplication size stacks                                             3 (U)
 DBCSR| Use memory pool for CPU allocation                                     F (U)
 DBCSR| Number of 3D layers                                               SINGLE (U)
 DBCSR| Use MPI memory allocation                                              F (U)
 DBCSR| Use RMA algorithm                                                      F (U)
 DBCSR| Use Communication thread                                               T (U)
 DBCSR| Communication thread load                                             87 (D)
 DBCSR| MPI: My process id                                                     0
 DBCSR| MPI: Number of processes                                              96
 DBCSR| OMP: Current number of threads                                         1
 DBCSR| OMP: Max number of threads                                             1
 DBCSR| Split modifier for TAS multiplication algorithm                  1.0E+00 (U)


  **** **** ******  **  PROGRAM STARTED AT               2024-07-08 08:01:18.270
 ***** ** ***  *** **   PROGRAM STARTED ON                                 c0266
 **    ****   ******    PROGRAM STARTED BY                              nichdsas
 ***** **    ** ** **   PROGRAM PROCESS ID                                868077
  **** **  *******  **  PROGRAM STARTED IN /scratch-grete/usr/nichdsas/test_pars
                                           er/cp2k-2024.1/smearing_need_added_mo
                                           s_medium

 CP2K| version string:                                       CP2K version 2024.1
 CP2K| source code revision number:                                             
 CP2K| cp2kflags: omp libint fftw3 libxc elpa parallel scalapack xsmm spglib sir
 CP2K|            ius libvori libbqb
 CP2K| is freely available from                            https://www.cp2k.org/
 CP2K| Program compiled at                                   2024-05-28 17:35:28
 CP2K| Program compiled on                                                      
 CP2K| Program compiled for                                               x86_64
 CP2K| Data directory path    /sw/rev/24.05/sapphirerapids_opa_rocky8/linux-rock
 CP2K| Input file name                                                 aiida.inp

 GLOBAL| Force Environment number                                              1
 GLOBAL| Basis set file name                                        BASIS_MOLOPT
 GLOBAL| Potential file name                                      GTH_POTENTIALS
 GLOBAL| MM Potential file name                                     MM_POTENTIAL
 GLOBAL| Coordinate file name                                      __STD_INPUT__
 GLOBAL| Method name                                                        CP2K
 GLOBAL| Project name                                                      aiida
 GLOBAL| Run type                                                         ENERGY
 GLOBAL| FFT library                                                       FFTW3
 GLOBAL| Diagonalization library                                            ELPA
 GLOBAL| DGEMM library                                                      BLAS
 GLOBAL| Minimum number of eigenvectors for ELPA usage                        16
 GLOBAL| Orthonormality check for eigenvectors                          DISABLED
 GLOBAL| Matrix multiplication library                                 ScaLAPACK
 GLOBAL| All-to-all communication in single precision                          F
 GLOBAL| FFTs using library dependent lengths                                  F
 GLOBAL| Grid backend                                                       AUTO
 GLOBAL| Global print level                                               MEDIUM
 GLOBAL| MPI I/O enabled                                                       T
 GLOBAL| Total number of message passing processes                            96
 GLOBAL| Number of threads for this process                                    1
 GLOBAL| This output is from process                                           0
 GLOBAL| Stack size for threads created by OpenMP (OMP_STACKSIZE)        default
 GLOBAL| CPU model name                           Intel(R) Xeon(R) Platinum 8468
 GLOBAL| CPUID                                                              1003

 MEMORY| system memory details [Kb]
 MEMORY|                        rank 0           min           max       average
 MEMORY| MemTotal            527678176     527678176     527678176     527678176
 MEMORY| MemFree             511247212     511247212     511247212     511247212
 MEMORY| Buffers                 12208         12208         12208         12208
 MEMORY| Cached                3706616       3706616       3706616       3706616
 MEMORY| Slab                  1392612       1392612       1392612       1392612
 MEMORY| SReclaimable           217228        217228        217228        217228
 MEMORY| MemLikelyFree       515183264     515183264     515183264     515183264


 *** Fundamental physical constants (SI units) ***

 *** Literature: B. J. Mohr and B. N. Taylor,
 ***             CODATA recommended values of the fundamental physical
 ***             constants: 2006, Web Version 5.1
 ***             http://physics.nist.gov/constants

 Speed of light in vacuum [m/s]                             2.99792458000000E+08
 Magnetic constant or permeability of vacuum [N/A**2]       1.25663706143592E-06
 Electric constant or permittivity of vacuum [F/m]          8.85418781762039E-12
 Planck constant (h) [J*s]                                  6.62606896000000E-34
 Planck constant (h-bar) [J*s]                              1.05457162825177E-34
 Elementary charge [C]                                      1.60217648700000E-19
 Electron mass [kg]                                         9.10938215000000E-31
 Electron g factor [ ]                                     -2.00231930436220E+00
 Proton mass [kg]                                           1.67262163700000E-27
 Fine-structure constant                                    7.29735253760000E-03
 Rydberg constant [1/m]                                     1.09737315685270E+07
 Avogadro constant [1/mol]                                  6.02214179000000E+23
 Boltzmann constant [J/K]                                   1.38065040000000E-23
 Atomic mass unit [kg]                                      1.66053878200000E-27
 Bohr radius [m]                                            5.29177208590000E-11

 *** Conversion factors ***

 [u] -> [a.u.]                                              1.82288848426455E+03
 [Angstrom] -> [Bohr] = [a.u.]                              1.88972613288564E+00
 [a.u.] = [Bohr] -> [Angstrom]                              5.29177208590000E-01
 [a.u.] -> [s]                                              2.41888432650478E-17
 [a.u.] -> [fs]                                             2.41888432650478E-02
 [a.u.] -> [J]                                              4.35974393937059E-18
 [a.u.] -> [N]                                              8.23872205491840E-08
 [a.u.] -> [K]                                              3.15774647902944E+05
 [a.u.] -> [kJ/mol]                                         2.62549961709828E+03
 [a.u.] -> [kcal/mol]                                       6.27509468713739E+02
 [a.u.] -> [Pa]                                             2.94210107994716E+13
 [a.u.] -> [bar]                                            2.94210107994716E+08
 [a.u.] -> [atm]                                            2.90362800883016E+08
 [a.u.] -> [eV]                                             2.72113838565563E+01
 [a.u.] -> [Hz]                                             6.57968392072181E+15
 [a.u.] -> [1/cm] (wave numbers)                            2.19474631370540E+05
 [a.u./Bohr**2] -> [1/cm]                                   5.14048714338585E+03
 

 CELL_TOP| Volume [angstrom^3]:                                        18.153110
 CELL_TOP| Vector a [angstrom     2.950     0.000     0.000   |a| =     2.950000
 CELL_TOP| Vector b [angstrom     1.475     2.555     0.000   |b| =     2.950000
 CELL_TOP| Vector c [angstrom     1.475     0.852     2.409   |c| =     2.950000
 CELL_TOP| Angle (b,c), alpha [degree]:                                60.000000
 CELL_TOP| Angle (a,c), beta  [degree]:                                60.000000
 CELL_TOP| Angle (a,b), gamma [degree]:                                60.000000
 CELL_TOP| Numerically orthorhombic:                                          NO
 CELL_TOP| Periodicity                                                       XYZ

 GENERATE|  Preliminary Number of Bonds generated:                             0
 GENERATE|  Achieved consistency in connectivity generation.

 CELL| Volume [angstrom^3]:                                            18.153110
 CELL| Vector a [angstrom]:       2.950     0.000     0.000   |a| =     2.950000
 CELL| Vector b [angstrom]:       1.475     2.555     0.000   |b| =     2.950000
 CELL| Vector c [angstrom]:       1.475     0.852     2.409   |c| =     2.950000
 CELL| Angle (b,c), alpha [degree]:                                    60.000000
 CELL| Angle (a,c), beta  [degree]:                                    60.000000
 CELL| Angle (a,b), gamma [degree]:                                    60.000000
 CELL| Numerically orthorhombic:                                              NO
 CELL| Periodicity                                                           XYZ

 CELL_REF| Volume [angstrom^3]:                                        18.153110
 CELL_REF| Vector a [angstrom     2.950     0.000     0.000   |a| =     2.950000
 CELL_REF| Vector b [angstrom     1.475     2.555     0.000   |b| =     2.950000
 CELL_REF| Vector c [angstrom     1.475     0.852     2.409   |c| =     2.950000
 CELL_REF| Angle (b,c), alpha [degree]:                                60.000000
 CELL_REF| Angle (a,c), beta  [degree]:                                60.000000
 CELL_REF| Angle (a,b), gamma [degree]:                                60.000000
 CELL_REF| Numerically orthorhombic:                                          NO
 CELL_REF| Periodicity                                                       XYZ

 *******************************************************************************
                                    Kpoints
 *******************************************************************************
 BRILLOUIN| K-point scheme                                        Monkhorst-Pack
 BRILLOUIN| K-Point grid                                             2    2    2
 BRILLOUIN| Accuracy in Symmetry determination                      0.100000E-05
 BRILLOUIN| K-Point point group symmetrization                               OFF
 BRILLOUIN| Wavefunction type                                            COMPLEX
 BRILLOUIN| List of Kpoints [2 Pi/Bohr]                                        4
 BRILLOUIN| Number           Weight            X              Y              Z
 BRILLOUIN|     1           0.25000       -0.25000       -0.25000       -0.25000
 BRILLOUIN|     2           0.25000       -0.25000       -0.25000        0.25000
 BRILLOUIN|     3           0.25000       -0.25000        0.25000       -0.25000
 BRILLOUIN|     4           0.25000       -0.25000        0.25000        0.25000
 *******************************************************************************

 *******************************************************************************
 *******************************************************************************
 **                                                                           **
 **     #####                         ##              ##                      **
 **    ##   ##            ##          ##              ##                      **
 **   ##     ##                       ##            ######                    **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####    **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##   **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######    **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##        **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##        **
 **           ##                                                    ##        **
 **                                                                           **
 **                                                ... make the atoms dance   **
 **                                                                           **
 **            Copyright (C) by CP2K developers group (2000-2024)             **
 **                      J. Chem. Phys. 152, 194103 (2020)                    **
 **                                                                           **
 *******************************************************************************

 *******************************************************************************
 *   ___                                                                       *
 *  /   \                                                                      *
 * [ABORT]                                                                     *
 *  \___/             Extra MOs (ADDED_MOS) are required for smearing          *
 *    |                                                                        *
 *  O/|                                                                        *
 * /| |                                                                        *
 * / \                                                   qs_environment.F:1553 *
 *******************************************************************************


 ===== Routine Calling Stack ===== 

            2 qs_init_subsys
            1 CP2K
